main.go

```diff
@@ -93,9 +95,14 @@ var providerNodepoolLabels = map[string]string{
 	"GCP":  "cloud.google.com/gke-nodepool",
 	"AKS":  "kubernetes.azure.com/agentpool",
 	"DOKS": "doks.digitalocean.com/node-pool-id",
+	"KOPS": "kops.k8s.io/instancegroup",
```

Comment:
This one doesn't look like is in the scope of this PR. If the idea is to support another predefined label I'd rather have it in a separated PR please.

Reply:
Good point, this is what I was using for testing, I will remove it for now since a custom label actually solves all the problems at once.